### DISCUSSION TITLE
Problemi installazione ArcadeDB (in italian)

### DISCUSSION THREAD

**Discussion by cr00589** [ArcadeData/arcadedb#234] [Q&A]
Buongiorno a tutti, spero qualcuno parli italiano dato che il mio inglese è pessimo.
Ho installato la versione Arcadedb 21.11.1, purtroppo non riesco a lanciare ne il server ne la console del software uso windows 10.
I seguenti file (server.bat e console.bat) si trovano nel seguente percorso:
C:\temp\arcadedb-21.11.1\package\src\main\scripts
Nella documentazione se non ho capito male dovrebbero essere nella cartella \bin ma questa non è presente da nessuna parte, mentre in Orientdb era presente.
Quando provo a lanciare ognuno degli file.bat indicati sopra ricevo il seguente errore:
C:\temp\arcadedb-21.11.1\package\src\main\scripts>server.bat
""
" ÔûêÔûêÔûêÔûêÔûêÔòù ÔûêÔûêÔûêÔûêÔûêÔûêÔòù  ÔûêÔûêÔûêÔûêÔûêÔûêÔòù ÔûêÔûêÔûêÔûêÔûêÔòù ÔûêÔûêÔûêÔûêÔûêÔûêÔòù ÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔòùÔûêÔûêÔûêÔûêÔûêÔûêÔòù ÔûêÔûêÔûêÔûêÔûêÔûêÔòù "
"ÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòùÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòùÔûêÔûêÔòöÔòÉÔòÉÔòÉÔòÉÔòØÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòùÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòùÔûêÔûêÔòöÔòÉÔòÉÔòÉÔòÉÔòØÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòùÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòù"
"ÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔòæÔûêÔûêÔûêÔûêÔûêÔûêÔòöÔòØÔûêÔûêÔòæ     ÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔòæÔûêÔûêÔòæ  ÔûêÔûêÔòæÔûêÔûêÔûêÔûêÔûêÔòù  ÔûêÔûêÔòæ  ÔûêÔûêÔòæÔûêÔûêÔûêÔûêÔûêÔûêÔòöÔòØ"
"ÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòæÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòùÔûêÔûêÔòæ     ÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòæÔûêÔûêÔòæ  ÔûêÔûêÔòæÔûêÔûêÔòöÔòÉÔòÉÔòØ  ÔûêÔûêÔòæ  ÔûêÔûêÔòæÔûêÔûêÔòöÔòÉÔòÉÔûêÔûêÔòù"
"ÔûêÔûêÔòæ  ÔûêÔûêÔòæÔûêÔûêÔòæ  ÔûêÔûêÔòæÔòÜÔûêÔûêÔûêÔûêÔûêÔûêÔòùÔûêÔûêÔòæ  ÔûêÔûêÔòæÔûêÔûêÔûêÔûêÔûêÔûêÔòöÔòØÔûêÔûêÔûêÔûêÔûêÔûêÔûêÔòùÔûêÔûêÔûêÔûêÔûêÔûêÔòöÔòØÔûêÔûêÔûêÔûêÔûêÔûêÔòöÔòØ"
"ÔòÜÔòÉÔòØ  ÔòÜÔòÉÔòØÔòÜÔòÉÔòØ  ÔòÜÔòÉÔòØ ÔòÜÔòÉÔòÉÔòÉÔòÉÔòÉÔòØÔòÜÔòÉÔòØ  ÔòÜÔòÉÔòØÔòÜÔòÉÔòÉÔòÉÔòÉÔòÉÔòØ ÔòÜÔòÉÔòÉÔòÉÔòÉÔòÉÔòÉÔòØÔòÜÔòÉÔòÉÔòÉÔòÉÔòÉÔòØ ÔòÜÔòÉÔòÉÔòÉÔòÉÔòÉÔòØ "
"PLAY WITH DATA                                    arcadedb.com"
The ARCADEDB_HOME environment variable is not defined correctly
This environment variable is needed to run this program

Ho provato ad impostare le varibili di ambiente in windows con il percorso C:\temp\arcadedb-21.11.1\package\src\main\scripts ma senza successo.
Ho provato a impostare il percorso C:\temp\arcadedb-21.11.1\package\src\main\scripts\bin copiandoci dentro i file .bat ma senza successo.
Ho confrontato gli script di ORIENTDB e ARCADEDB aprendoli con un notepad e non ho visto differenze, ORIENTDB però funziona.
Non so proprio più cosa fare, spero che qualcuno mi possa dare la soluzione corretta, voglio imparare i database grafici per alcuni progetti che ho in mente.
Ringrazio in anticipo chiunque provi a darmi una risposta.

**Comment by lvca** [comment]
Ciao, se parti dai sorgenti devi compilare il tutto, altrimenti fai prima a prendere le release gia' compilate. Questa e' l'ultima:

https://github.com/ArcadeData/arcadedb/releases/tag/21.11.1

**Reply by cr00589** [reply]
Grazie infinite per la dritta Luca sei stato gentilissimo, adesso proverò a vedere se riesco a creare un database nuovo, popolarlo con un file csv, alcuni dati te li avevo mandati tramite mail qualche giorno fà e poi devo vedere se riuscirò ad interrogare il database in modo corretto. Spero di poter condividere con te una esperienza positiva...credo molto nelle possibilità dei database a grafo....ti terrò aggiornato sugli sviluppi...grazie 1000 davvero per la disponibilità.
